App.js

```diff
@@ -2,11 +2,22 @@ import React from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 
 export default class App extends React.Component {
+  state = {
+    fontsLoaded: false,
+  }
+  async componentWillMount() {
+    await Expo.Font.loadAsync({
+      'Roboto': require('native-base/Fonts/Roboto.ttf'),
+      'Roboto_medium': require('native-base/Fonts/Roboto_medium.ttf'),
+    });
+
+    this.setState({ fontsLoaded: true });
+  }
+
   render() {
+    if (!this.state.fontsLoaded) return null;
     return (
-      <View style={styles.container}>
-        <Text>Werewolf</Text>
-      </View>
+      null
```

Comment:
I think I'll need some explanation here for me to understand. 🙈

Reply:
I'm rendering null because by the moment the app has no routing, so I put the component in progress there. This will be updated in #12